TASK 104 Fetch old quizzes frontend implementation

roosavihlman opened the pull request:
Ticket: https://github.com/orgs/APUOPE-RE/projects/3/views/1?pane=issue&itemId=89056158&issue=APUOPE-RE%7Cfrontend%7C104

Implemented previous quiz fetch

Not the prettiest, let's refactor later

mohes7489 commented:
few issues:
- search bar not functioning.
- multiple warning in the console about quizzes having the same key.. I am not sure if this is an issue
- sometimes the score not available
- sometimes the previous quizzes column is not showing anything, until I refresh the page or switch between tabs.. maybe we limit the number of quizzes being fetched from the backend?
otherwise, the functionality look good :)

roosavihlman commented:
> few issues:
> 
> * search bar not functioning.
> * multiple warning in the console about quizzes having the same key.. I am not sure if this is an issue
> * sometimes the score not available
> * sometimes the previous quizzes column is not showing anything, until I refresh the page or switch between tabs.. maybe we limit the number of quizzes being fetched from the backend?
>   otherwise, the functionality look good :)

Fixed the score to be seen even though score is zero. Couldn't reproduce the key element warning since all items have keys, but I removed few console logs that logged ids. Didn't touch the search bar since it wasn't part of the ticket. Good future implementation ticket to be done though.
